src/instaskip/actions/paths.clj

```diff
@@ -0,0 +1,28 @@
+(in-ns 'instaskip.actions)
+
+(defn list-paths [filters innkeeper-config]
+
+  (let [paths (match filters
+                     {:team team}
+                     ; =>
+                     (ik/get-paths {:owned-by-team team} innkeeper-config)
+
+                     :else
+                     ; =>
+                     (ik/get-paths innkeeper-config))
+
+        paths-with-hosts (->> paths
+                              (map (fn [{:keys [id uri]}]
```

Comment:
This code is doing destructuring. `id` and `uri` get bound to the map for the specified keys. `select-keys` would only remove the unwanted keys from the map.